scripts/src/models/users.ts

```diff
@@ -181,6 +181,10 @@ export class Wallets {
 		return this.items.length;
 	}
 
+	public get uniqueCount() {
```

Comment:
why do we need both?
why not just change the `count` getter to return the unique count?

Reply:
a. `count` sounds like a method we might use.
but mostly b. Not sure it's not used already in other places (amd wasn't too bothered to search for it.)

Reply:
@nitzantomer Since it's pretty generic and straightforward implementation I would leave it be but if you want me to remove it and sure it's not used elsewhere, I will. Just let me know, I want to merge this.

Reply:
It is used in ```Wallets.lastUsed()``` so I'm leaving it as it is. Can you approve this PR now?

Reply:
@liorama 
but the idea behind `count` was to return a unique count, it was a bug that it didn't.
the fact that it's being used is irrelevant, as it uses a bugged function.
there's no reason to have to different getters, especially since one of them doesn't function as it should have.

Reply:
Ok got you, done, I thought it was the intended behaviour for other uses.

Reply:
@nitzantomer